bin/shoryuken

```diff
@@ -35,7 +35,12 @@ module Shoryuken
         opts[:config_file] = opts.delete(:config) if opts[:config]
 
         # Keep compatibility with old CLI queue format
-        opts[:queues] = options[:queues].map { |q| q.split(',') } if options[:queues]
+        if options[:queues]
+          opts[:queues] = options[:queues].map do |q|
+            queue_name, batch_size = q.split(',')
+            [queue_name, batch_size.to_i]
```

Comment:
@matrinox isn't [this code](https://github.com/phstc/shoryuken/blob/bbfa813939f88e4e64beb178df85d9868bbff15f/lib/shoryuken/environment_loader.rb#L120) doing the `to_i` already?